ESMF/GOCART2G_GridComp/GOCART2G_GridCompMod.F90

```diff
@@ -144,20 +144,20 @@ subroutine SetServices (GC, RC)
 !   Retrieve wavelengths from GOCART2G_GridComp.rc
     n_wavelengths_profile = ESMF_ConfigGetLen (myCF, label='wavelengths_for_profile_aop_in_nm:', __RC__)
     n_wavelengths_vertint = ESMF_ConfigGetLen (myCF, label='wavelengths_for_vertically_integrated_aop_in_nm:', __RC__)
-    n_wavelengths_diagmie = ESMF_ConfigGetLen (myCF, label='aerosol_monochromatic_optics_wavelength:', __RC__)
+    n_wavelengths_diagmie = ESMF_ConfigGetLen (myCF, label='aerosol_monochromatic_optics_wavelength_in_nm_from_LUT:', __RC__)
 
     allocate(self%wavelengths_profile(n_wavelengths_profile), self%wavelengths_vertint(n_wavelengths_vertint), &
              wavelengths_diagmie(n_wavelengths_diagmie), __STAT__)
 
     call ESMF_ConfigGetAttribute (myCF, self%wavelengths_profile, label='wavelengths_for_profile_aop_in_nm:', __RC__)
     call ESMF_ConfigGetAttribute (myCF, self%wavelengths_vertint, label='wavelengths_for_vertically_integrated_aop_in_nm:', __RC__)
-    call ESMF_ConfigGetAttribute (myCF, wavelengths_diagmie, label='aerosol_monochromatic_optics_wavelength:', __RC__)
+    call ESMF_ConfigGetAttribute (myCF, wavelengths_diagmie, label='aerosol_monochromatic_optics_wavelength_in_nm_from_LUT:', __RC__)
 
 !   Set wavelengths in universal config
 
     call MAPL_ConfigSetAttribute (cf, self%wavelengths_profile, label='wavelengths_for_profile_aop_in_nm:', __RC__)
     call MAPL_ConfigSetAttribute (cf, self%wavelengths_vertint, label='wavelengths_for_vertically_integrated_aop_in_nm:', __RC__)
-    call MAPL_ConfigSetAttribute (cf, wavelengths_diagmie, label='aerosol_monochromatic_optics_wavelength:', __RC__)
+    call MAPL_ConfigSetAttribute (cf, wavelengths_diagmie, label='aerosol_monochromatic_optics_wavelength_in_nm_from_LUT:', __RC__)
```

Comment:
`wavelengths_diagmie` is still declared as `integer`: https://github.com/GEOS-ESM/GOCART/blob/58af7747eba91dc597c65485a5e4e781cb25deca/ESMF/GOCART2G_GridComp/GOCART2G_GridCompMod.F90#L113
which is inconsistent with `self%wavelengths_profile` and `self%wavelengths_vertint` (`real`).

I would still recommend including changes from PR #65 into this branch to address this issue and streamline the code. For example, conversion from `nm` to `m` should be done only once (see https://github.com/GEOS-ESM/GOCART/pull/65/files#diff-682724d291af0ad213c4e1b2ffa97156de4989543290b60bf362adfa98e3d8da):

https://github.com/GEOS-ESM/GOCART/blob/b9fa4c6c0ed46e43dae1a66aaa296a0d72ff839f/ESMF/GOCART2G_GridComp/GOCART2G_GridCompMod.F90#L156-L162

Reply:
@rmontuoro I guess they are declared integers because they are integers in the file:
https://github.com/GEOS-ESM/GOCART/blob/58af7747eba91dc597c65485a5e4e781cb25deca/ESMF/GOCART2G_GridComp/GOCART2G_GridComp.rc#L39

But as you said, the others aren't. For now, we might accept this as is and then you or I can make a (new) PR to fix it once we have a good "baseline" to compare to.